src/components/utils/include/utils/qdb_wrapper/sql/sql_database.h

```diff
@@ -119,6 +120,12 @@ class SQLDatabase {
    */
   Error error_;
 
+  /**
+   *  The temporary in-memory database
+   *  @see SQLite manual
+   */
```

Comment:
@BSolonenko  we can keep it only in cc file

Reply:
@LuxoftAKutsan fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/4154b9b3932adce8017f71b9fa54d0da8aa9e499